map_model/src/make/buildings.rs

```diff
@@ -369,12 +369,28 @@ fn classify_bldg(
         return BuildingType::Commercial;
     }
 
-    if tags.is("key", "value") {
-        return BuildingType::Residential(42);
+    if tags.is("building", "office") {
+        return BuildingType::Commercial;
+    }
+
+    // TODO - use some "in list" to avoid duplicating
```

Comment:
```
    fn is_any(&self, k: &str, values: Vec<&str>) -> bool {
        if let Some(v) = self.0.get(k) {
            values.contains(&v.as_ref())
        } else {
            false
        }
    }                                                                                      
                                                                                                       
    tags.is_any("key", vec!["v1", "v2", "v3"])
```

Reply:
Thanks, I think that I now managed to use it correctly.